.github/workflows/release.yml

```diff
@@ -39,22 +51,106 @@ jobs:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
+      -
+        name: "Build and push ${{ matrix.arch }}"
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./build/dockerfiles/Dockerfile
+          target: registry
+          build-args: |
+            PATCHED_IMAGES_TAG=${{ github.event.inputs.version }}
+          platforms: linux/${{ matrix.arch }}
+          push: true
+          tags: ${{ env.IMAGE }}:${{ github.event.inputs.version }}-${{ matrix.arch }}
+      -
+        id: result
+        name: "Build result outputs version"
+        if: ${{ success() }}
+        run: echo "::set-output name=${{ matrix.arch }}::${{ github.event.inputs.version }}-${{ matrix.arch }}"
+  
+  create-manifest:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: "Checkout source code"
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
       - 
-        name: Login to docker.io
+        name: Prepare
+        id: prep
+        run: |
+          SHORT_SHA1=$(git rev-parse --short HEAD)
+          echo ::set-output name=short_sha1::${SHORT_SHA1}
+      -
+        name: "Docker quay.io Login"
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
```

Comment:
I believe this should be logging into docker.io, to prevent rate limiting

Reply:
As we are not performing push/pull to dockerhub, I had removed docker.io login. Should I add login to docker.io?

Reply:
Yes, I believe we need it as the build process will pull from dockerhub. We might run into rate limiting because of this, so it's important to login to docker.io

Reply:
okay, will add docker login back.